index.src.html

```diff
@@ -660,8 +883,104 @@
               "#def-constraint-focalLengthY">focalLengthY</a></code> for
               details.
             </dd>
+            <dt>
+              <dfn><code>principalPointX</code></dfn> of type <span class=
+              "idlMemberType"><a>double</a></span>
+            </dt>
+            <dd>
+              See <code><a href=
+              "#def-constraint-principalPointX">principalPointX</a></code> for
+              details.
+            </dd>
+            <dt>
+              <dfn><code>principalPointY</code></dfn> of type <span class=
+              "idlMemberType"><a>double</a></span>
+            </dt>
+            <dd>
+              See <code><a href=
+              "#def-constraint-principalPointY">principalPointY</a></code> for
+              details.
+            </dd>
+            <dt>
+              The <dfn><code>deprojectionDistortionCoefficients</code></dfn>
+              dictionary member represents the media track's (usually the
+              <a>depth map</a>'s) <a>distortion coefficients</a> used when
+              deprojecting from 2D to 3D space.
+            </dt>
+            <dd>
+              See <code><a href=
+              "#def-constraint-deprojectionDistortionCoefficients">deprojectionDistortionCoefficients</a></code>
+              for details.
+            </dd>
+            <dt>
+              The <dfn><code>projectionDistortionCoefficients</code></dfn>
+              dictionary member represents the media tracks's <a>distortion
+              coefficients</a> used when projecting from 3D to 2D space.
+            </dt>
+            <dd>
+              See <code><a href=
+              "#def-constraint-projectionDistortionCoefficients">projectionDistortionCoefficients</a></code>
+              for details.
+            </dd>
+            <dt>
+              The <dfn><code>depthToVideoTransform</code></dfn> dictionary
+              member represents the <a>depth map</a>'s camera's
+              <a>transformation from depth to video</a> camera 3D coordinate
+              system.
+            </dt>
+            <dd>
+              See <code><a href=
+              "#def-constraint-depthToVideoTransform">depthToVideoTransform</a></code>
+              for details.
+            </dd>
           </dl>
         </section>
+        <h3>
+          <dfn><code>DistortionCoefficients</code></dfn> dictionary
+        </h3>
+        <div data-dfn-for="DistortionCoefficients">
+          <dl class="dictionary-members">
+            <dt>
+              The <dfn>k1</dfn>, <dfn>k2</dfn>, <dfn>p1</dfn>, <dfn>p2</dfn>
```

Comment:
Sentence added:
              They are used when there's need to <a>deproject</a> depth value to 3D space or to <a>project</a> 3D value to 2D video frame coordinates.

I feel we will need to add examples for the cases listed by @bbernhar in #110 but that would need to come later.